recipes/pantax/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pantax" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/LuoGroup2023/PanTax/releases/download/v{{ version }}/{{ name }}.tar.gz  
+    sha256: c14a46b41f580cf39ad502049d267cd31bbfd4dc05028215cc669c0e52f58ce7
+
+build:
+  skip: True  # [osx]
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}   
```

Comment:
`"x.x"` here means that this package may introduce API breaks in minor versions. It is usually used for `0.x` versions. Are you sure you want `"x.x"` instead of `"x"` ?

Reply:
I have already made modifications.